### ISSUE TITLE
Create a command to extract the French bank registry

### ISSUE THREAD

**Issue by Natim** [mdomke/schwifty#173]
What we are looking for is this list: https://www.regafi.fr/spip.php?page=results&type=advanced&id_secteur=1&lang=fr&denomination=&siren=&cib=&bic=&nom=&siren_agent=&num=&cat=21-TBR07&retrait=0

**Comment by Natim** [comment]
Maybe what we need is to get access to the FIB: https://www.banque-france.fr/en/node/2661

**Comment by Natim** [comment]
I found a listing here: https://github.com/sepinf-inc/IPED/blob/master/iped-engine/src/main/resources/swift/FR.csv
But 3 years old is a bit too old to have the last neobanks.

**Comment by mdomke** [comment]
@Natim In general, what we are looking for is some kind of translation between national bank code and BIC. In the first link, I can find the bank code but not the BIC and in the last link there is a BIC but no bank code, or am I missing something? For the Banque de France link, I don't fully understand how to get access to the information. Thank you for your effort so far, however!

**Comment by Natim** [comment]
Yes, the information is not easy to access, I'm still trying to figure something out regarding this.
